crates/sdk/src/callbacks.rs

```diff
@@ -51,34 +51,34 @@ impl CallbackId {
 /// Manages row callbacks for a `DbContext`/`DbConnection`.
 pub struct DbCallbacks<M: SpacetimeModule> {
     /// Maps table name to a set of callbacks.
-    table_callbacks: HashMap<&'static str, TableCallbacks<M>>,
+    table_callbacks: IntMap<u32, TableCallbacks<M>>,
```

Comment:
Comment is no longer correct. It's also not clear what the actual key is, now - is it the table ID received during handshake, or the table index computed at codegen time?